config/spryk/spryks/Spryker/Glue/BackendApi/Plugin/AddGlueBackendApiResourcePlugin.yml

```diff
@@ -0,0 +1,90 @@
+spryk: template
+description: "Adds a new GlueBackendApi Resource Plugin."
+mode: both
+
+arguments:
+    module:
+        inherit: true
+
+    organization:
+        inherit: true
+        default: Spryker
+
+    resource:
+        description: "Defines the resource name this resource plugin will work with."
+
+    className:
+        value: "{{ resource }}Resource"
+
+    extends:
+        value: \Spryker\Glue\GlueApplication\Plugin\GlueApplication\AbstractResourcePlugin
+
+    implements:
+        value: \Spryker\Glue\GlueRestApiConventionExtension\Dependency\Plugin\RestResourceInterface
+
+    template:
+        value: Common/Model.php.twig
+
+    targetFilename:
+        value: "{{ resource }}Resource.php"
+
+    targetPath:
+        value: "src/{{ organization }}/Glue/{{ module }}/Plugin/"
+
+    application:
+        value: Glue
+
+    layer:
```

Comment:
Where is that used and why it's named `layer`?

Reply:
Used in the namespace in the model template. Should I use subDirectory instead?

Reply:
Yes please as we do with other places as well.